<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>端口流量监控系统</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_5">
     <property name="styleSheet">
      <string notr="true">QRadioButton{

	font-family: &quot;Microsoft YaHei&quot;;
font-size: 18px;
font-weight: bold;
color: rgb(0, 170, 255);
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QRadioButton" name="rbtn1">
      <property name="text">
       <string>图形概览</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn2">
      <property name="text">
       <string>网络流量</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn3">
      <property name="text">
       <string>网络设备</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn4">
      <property name="text">
       <string>提醒</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stack_1">
     <property name="styleSheet">
      <string notr="true">.QStackedWidget{
	background-color: rgb(255, 255, 255);
	
}</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>全部</string>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_1">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>应用程序</string>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_2"/>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_12">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>计算机性能</string>
           </property>
          </widget>
          <widget class="QTextBrowser" name="textBrowser_6"/>
         </widget>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label_5">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>网络设备</string>
           </property>
          </widget>
          <widget class="QTextBrowser" name="textBrowser_3"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QRadioButton" name="rbtn7">
            <property name="text">
             <string>全部</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="rbtn8">
            <property name="text">
             <string>未读</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLCDNumber" name="LCD"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QStackedWidget" name="stack_3">
           <property name="styleSheet">
            <string notr="true">*{
	background-color:rgb(255, 253, 228)
}</string>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page_5">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QTextBrowser" name="textBrowser_4">
               <property name="styleSheet">
                <string notr="true">QTextBrowser{
	background-color:rgb(255, 255, 255)
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_6">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QListView" name="listView_2"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(209, 209, 209);
border-radius:6px;
</string>
      </property>
      <widget class="QSplitter" name="splitter_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>751</width>
         <height>31</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rbtn5">
           <property name="text">
            <string>全部</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtn6">
           <property name="text">
            <string>应用程序</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 16px;
	font-weight: bold;
	color: rgb(0, 0, 0);
}
	</string>
        </property>
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stack_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>1031</width>
         <height>571</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(227, 227, 227);






</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_7">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QPushButton" name="button1">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>530</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 background-color:rgb(255, 0, 127)
font: 700 12pt &quot;Microsoft YaHei UI&quot;;
}</string>
         </property>
         <property name="text">
          <string>图像展示</string>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>991</width>
           <height>481</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">*{
	background-color:white;
	font: 12pt &quot;Microsoft YaHei UI&quot;
}</string>
          </property>
          <property name="text">
           <string>流量图</string>
          </property>
         </widget>
         <widget class="MplWidget2" name="asss" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_8">
        <widget class="QGraphicsView" name="graphicsView">
         <property name="geometry">
          <rect>
           <x>245</x>
           <y>33</y>
           <width>771</width>
           <height>521</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">*{
	background-color:rgb(255, 255, 255)
}</string>
         </property>
        </widget>
        <widget class="QListView" name="listView">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>33</y>
           <width>221</width>
           <height>521</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">*{
	blackground-color: rgb(255, 255, 255);
	font: 700 7pt &quot;Microsoft JhengHei UI&quot;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>10</y>
           <width>48</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel#label_6{
	font: 12pt &quot;Modern No. 20&quot;
}</string>
         </property>
         <property name="text">
          <string>流量图</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget2</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
